src/server/game/DungeonFinding/LFGQueue.cpp

```diff
@@ -429,8 +432,27 @@ namespace lfg
             data.group = proposalGroups.find(itRoles->first)->second;
             if (!proposal.isNew && data.group && data.group == proposal.group) // Player from existing group, autoaccept
                 data.accept = LFG_ANSWER_AGREE;
+
+            if (!completedEncounters && !proposal.isNew)
+            {
+                if (LFGDungeonEntry const* dungeon = sLFGDungeonStore.LookupEntry(proposal.dungeonId))
+                {
+                    if (Player* player = ObjectAccessor::FindConnectedPlayer(itRoles->first))
+                    {
+                        if (player->GetMapId() == dungeon->map)
```

Comment:
fatal error: comparison of integers of different signs: 'uint32' (aka 'unsigned int') and 'const int32' (aka 'const int') [-Wsign-compare]

Reply:
@UltraNix @faq has this been solved?

Reply:
Yes